rebar.config

```diff
@@ -18,7 +18,8 @@
 
 {deps, [
         {jsx, "2.9.0"},
-        {lhttpc, "1.6.2"},
+        {lhttpc, ".*",
+            {git, "https://github.com/erlcloud/lhttpc", {tag, "1.6.2"}}},
```

Comment:
Why did we need to change the format here?

Reply:
Probably to address a license scout issue, iirc.